app/src/main/java/lc/wise/finenancer/presentation/assets/details/AssetDetailsViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package lc.wise.finenancer.presentation.assets.details
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import lc.wise.finenancer.data.stub.repository.StubAssetRepository
+import lc.wise.finenancer.domain.entity.Asset
+
+@HiltViewModel
+class AssetDetailsViewModel @Inject constructor(
+    private val stubAssetRepository: StubAssetRepository
```

Comment:
```suggestion
    private val assetRepository: AssetRepository
```

Зачем нам тут зависеть от конкретной реализации? У тебя есть DI, который подсунет нужную реализацию